functions

```diff
@@ -135,7 +135,10 @@ service_import() {
   if [[ -t 0 ]]; then
     dokku_log_fail "No data provided on stdin."
   fi
-  docker exec -i "$SERVICE_NAME" mysql --user=root --password="$ROOTPASSWORD" "$DATABASE_NAME"
+
+  docker exec "$SERVICE_NAME" bash -c "printf '[client]\nuser=root\npassword=$ROOTPASSWORD\n' > /root/credentials.cnf"
+  docker exec -i "$SERVICE_NAME" mysql --defaults-extra-file=/root/credentials.cnf "$DATABASE_NAME"
+  docker exec "$SERVICE_NAME" rm /root/credentials.cnf
```

Comment:
Why this particular change?

Reply:
This change is introduced to surpress warnings from mysql
> Warning: Using a password on the command line interface can be insecure.

https://stackoverflow.com/questions/20751352/suppress-warning-messages-using-mysql-from-within-terminal-but-password-written